clients/src/main/java/org/apache/kafka/clients/admin/RemoveMembersFromConsumerGroupOptions.java

```diff
@@ -31,13 +32,13 @@
 @InterfaceStability.Evolving
 public class RemoveMembersFromConsumerGroupOptions extends AbstractOptions<RemoveMembersFromConsumerGroupOptions> {
 
-    private Set<MemberToRemove> members;
+    private Set<LeaveGroupRequestData.MemberIdentity> members;
 
-    public RemoveMembersFromConsumerGroupOptions(Collection<MemberToRemove> members) {
+    public RemoveMembersFromConsumerGroupOptions(Collection<LeaveGroupRequestData.MemberIdentity> members) {
```

Comment:
This is an breaking public API change -- we can't allow this.

Reply:
Thanks ,mjsax !
About the public API change, Boyang, Chen has also replied in the related JIRA, let me go through it first and will update soon. :)